sys/include/net/gnrc/ipv6/nib.h

```diff
@@ -288,6 +288,19 @@ enum {
     GNRC_IPV6_NIB_ROUTE_INFO_TYPE_NSC,
 };
 
+/**
+ * @brief   Types for GNRC_NETIF_BUS_NIB per-interface message bus
+ */
+typedef enum {
+    /**
+     * @brief   Address registration successful
+     *
+     * The Address registration with the boarder router was successful.
```

Comment:
1.

```suggestion
     * The Address registration with the border router was successful.
```

2. This comment is misleading, as registration only happens to the upstream router in normal configuration. For registration with the border router, [multihop DAD](https://tools.ietf.org/html/rfc6775#section-8.2) is required, which currently is not implemented.